crates/wast/src/core/binary.rs

```diff
@@ -342,6 +342,18 @@ impl From<&ArrayType<'_>> for wasm_encoder::ArrayType {
     }
 }
 
+impl Encode for ContType<'_> {
+    fn encode(&self, e: &mut Vec<u8>) {
+        self.0.encode(e);
+    }
+}
```

Comment:
If you'd like it should be ok to remove this now as it's no longer needed (just a minor thing though)